sources/php5/KalturaClientBase.php

```diff
@@ -706,7 +706,7 @@ public function setClientConfiguration(KalturaClientConfiguration $configuration
 		$params = get_class_vars('KalturaClientConfiguration');
 		foreach($params as $param)
 		{
-			if(is_null($configuration->$param))
+			if(empty($configuration->$param) || is_null($configuration->$param))
```

Comment:
We might have properties in the future that may accept empty strings and it will be reasonable.
Why is that change needed?